codalab/rest/bundles.py

```diff
@@ -343,22 +345,36 @@ def _fetch_bundle_contents_info(uuid, path=''):
           "link": "<string representing target if file is a symbolic link>",
           "type": "<file|directory|link>",
           "size": <size of file in bytes>,
-          "contents": {
-            "name": ...,
-            <contents of the directory represented recursively with the same schema>
-          },
-          "perm", <unix permission integer>
+          "perm": <unix permission integer>,
+          "contents": [
+              {
+                "name": ...,
+                <each file of directory represented recursively with the same schema>
+              },
+              ...
+          ]
       }
     }
     ```
     """
     depth = query_get_type(int, 'depth', default=0)
+    human_readable = query_get_bool('human_readable', default=False)
     if depth < 0:
         abort(httplib.BAD_REQUEST, "Depth must be at least 0")
 
     check_bundles_have_read_permission(local.model, request.user, [uuid])
+    info = local.download_manager.get_target_info(uuid, path, depth)
+
+    def render_human_readable(target_info):
+        if 'size' in target_info:
+            target_info['size'] = formatting.size_str(target_info['size'])
```

Comment:
Not sure I like multiplexing between these two values...can we always just return two fields, `size` and `size_str` and not even have the flag?